nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java

```diff
@@ -761,14 +791,39 @@ public String getVendor() {
         return "rs2d";
     }
 
+    @Override
+    public boolean arePhasesSet(int dim) {
+        double ph0 = phases[dim][0];
+        double ph1 = phases[dim][1];
+        return Math.abs(ph0) > 1.0e-9 || Math.abs(ph1) > 1.0e-9;
+    }
+
     @Override
     public double getPH0(int dim) {
-        return 0.0;
+        double ph0 = phases[dim][0];
+        // phase is reversed between RS2D and NMRfx
+        return convertRS2DPhase0ToNMRFx(ph0, dim);
+    }
+
+    private double convertNMRFxPhase0ToRS2D(double phase, int dim) {
+        phase = -phase;
+        if (dim == 0) {
+            phase += 90;
+        }
+        return phase;
+    }
+
+    private double convertRS2DPhase0ToNMRFx(double phase, int dim) {
+        if (dim == 0) {
+            phase -= 90; // empirical
+        }
+        return -phase;
```

Comment:
Copied how the JCAMP was reading PH0 and PH1, seemed like this was right when comparing to screenshots from data from JCAMP export/2022-07-07